### ISSUE TITLE
passing a Buffer to waterline-sequel generates an incorrect query

### ISSUE THREAD

**Issue by kevinburkeshyp** [balderdashy/waterline-sequel#61]
Steps to reproduce:

- Create a column with type `BYTEA` in Postgres, say, `passwordHash`
- You can't set it to type: `'bytea'`in Waterline; it fails anchor validation in Waterline. Instead you have to use `type: 'binary'`
- You can't pass strings as the value, even `'binary'` encoded ones. it fails anchor validation with the cryptic string `'[object Object]'`
- Anyway you finally manage to write data by passing a Buffer as the column value - `{passwordHash: new Buffer('foo bar') }`. The data looks like it got written properly!
- you try to retrieve that data using `Users.find({passwordHash: new Buffer('foo bar')});`
- Waterline-sequel (specifically, a call to `_.cloneDeep`) transforms this Buffer into a dictionary with many keys. I've reproduced this behavior in a gist here: https://gist.github.com/kevinburkeshyp/c45a344a15e040ec6f0e

Sorry I don't have time to find out exactly what's breaking - decided to bail on this and just use base64 and text instead.

**Comment by sailsbot** [comment]
Thanks for posting, @kevinburkeshyp.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
